Makefile

```diff
@@ -169,7 +166,7 @@ bin/$(TARGET_OS)/$(TARGET_ARCH):
 
 bin/$(TARGET_OS)/$(TARGET_ARCH)/manager: $(GO_SOURCES) bin/$(TARGET_OS)/$(TARGET_ARCH)
 	@echo "Building operator with version $(VERSION); $(TARGET_OS) - $(TARGET_ARCH)"
-	CGO_ENABLED=0 GOOS=$(TARGET_OS) GOARCH=$(TARGET_ARCH) go build -o $@ -ldflags="-X $(MODULE_PATH)/pkg/version.Version=$(VERSION)" cmd/manager/main.go
+	CGO_ENABLED=0 GOOS=$(TARGET_OS) GOARCH=$(TARGET_ARCH) go build -o $@ -ldflags="-X github.com/mongodb/mongodb-atlas-kubernetes/v2/pkg/version.Version=$(VERSION)" cmd/manager/main.go
```

Comment:
That's cool that we can put this back in here. How do we ensure though that old builds still work?

Reply:
Because this change will use the build mechanism of each release, as it will NOT overwrite any of the files from the old branch:
- Old releases without the `fast.Dockerfile` will use the regular `Dockerfile` and build as they used to (quite slow BTW).
- New releases with `fast.Dockerfile` will use it.
- Newest releases with `make `check-version` target in the `Makefile` will also include that check.

Reply:
perfect, thank you for the rationale!